src/bitcoin-cli.cpp

```diff
@@ -569,6 +569,56 @@ class NetinfoRequestHandler : public BaseRequestHandler
         "> bitcoin-cli -netinfo help\n"};
 };
 
+/** Process addressinfo requests */
+class AddressinfoRequestHandler : public BaseRequestHandler
+{
+private:
+    static constexpr uint8_t m_networks_size{5};
+    const std::array<std::string, m_networks_size> m_networks{{"ipv4", "ipv6", "torv2", "torv3", "i2p"}};
+    std::array<uint64_t, m_networks_size> m_counts{{}}; // Address counts by network
+    uint64_t m_total{0}; // Total address count
```

Comment:
57eb71ce0f27c0353ea0742a696c6d04ce048e15

No need to have `m_networks` and `m_total` as members.

Reply:
Changed `m_counts` and `m_total` to be local variables instead (`counts` and `total`).